apis/placement/v1beta1/commons.go

```diff
@@ -5,13 +5,6 @@ Licensed under the MIT license.
 
 package v1beta1
 
-import (
-	v1 "k8s.io/api/core/v1"
-	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
-)
-
-type ClusterState string
-
 const (
 	// Unprefixed labels/annotations are reserved for end-users
 	// we will add a kubernetes-fleet.io to designate these labels/annotations as official fleet labels/annotations.
```

Comment:
update the fleetPrefix as "placement.kubernetes-fleet.io/"?

Reply:
I was thinking this is the common prefix for all.

Reply:
that works too, need to move this common prefix out of the apis/placement/beta1/common.go to  a common place, eg, apis/common.go?